### PR TITLE
Update leaves-of-absence/README.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Updated LOA page to include entity specific information re: C19

### CODE REVIEW

**Review by justinegeffen — DISMISSED** [review]
LGTM. Note: I've committed editorial changes to the content.

File: operations/workplace/people/working-at-mattermost/leaves-of-absence/README.md [hunk 1]
```diff
@@ -1,37 +1,51 @@
...
-5. **Notify** [**HR**](mailto:%20hr@mattermost.com) **via email if the effective Leave of Absence date changes** **from the initial BambooHR request.** An expected baby due date is likely to change. In order to minimize disruption in pay for the Staff member on leave, managers are required to notify HR of any changes to the initial Leave Request.
-
- 
+1. **Inform and meet with HR.** Every region has specific statutory sick and family leave requirements. Each situation may be different in eligibility criteria andd benefit amounts. 
+2. **Finalize the dates of leave.** Work with the staff member who is planning a leave. This should be completed at least 30 days prior to the leave date, when possible. 
+3. **Approve dates in Bamboo.** Especially in instances where a birth date is not planned, the dates will be approximate and expected to adjust accordingly.
+4. **Create and communicate the plan for coverage. Managers should work with the staff member to create a coverage plan and identify training and documentation that will ensure responsibilities** [are covered](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check).
```
Comment (TQuock) [inline operations/workplace/people/working-at-mattermost/leaves-of-absence/README.md]:
```suggestion
4. **Create and communicate the plan for coverage.** Managers should work with the staff member to create a coverage plan and identify training and documentation that will ensure responsibilities [are covered](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check).
```

**Review by TQuock — DISMISSED** [review]
The content is approved. I made one minor editorial change that needs to be reviewed by @justinegeffen for consistency.

**Review by jasonblais — APPROVED** [review]

**Comment by jasonblais** [comment]
Reviewed by SME (Tim) and editor (Justine), merging

**Pull request merged**